docs/UserGuide.md

```diff
@@ -107,15 +165,25 @@ Allows you to add a guest to WedLog, keeping track of important details such as
 
 Format: `guest add n/NAME [p/PHONE] [e/EMAIL] [a/ADDRESS] [r/RSVP_STATUS] [tn/TABLE_NUMBER] [d/DIETARY_REQUIREMENT]… [t/TAG]…`
 
->Tips:
-><br>
->- Items in square brackets are optional.
->  <br>
->- A guest can have any number of dietary requirements and tags (including 0).
->  <br>
->- A person can be recorded as both a guest and vendor.
->(e.g. Even if there is a vendor named `Gia`, you are also able to add a guest named `Gia`)
->- Refer to [Appendix A](#7-1-appendix-a-acceptable-values-for-parameters) for more details on the acceptable values for the parameters.
+<div class="alert alert-block alert-warning">
+
+**Parameter information:**
+
+* Parameters in square brackets are optional.
+* A guest can have any number of dietary requirements and tags (including 0). 
+
+Refer to [Appendix A](#7-1-appendix-a-acceptable-values-for-parameters) for more details on the acceptable values for the parameters.
+
+</div>
+
+<div class="alert alert-info">
+
+**Note:**
+
+* A person can be recorded as both a guest and vendor. (e.g. Even if there is a vendor named `Gia`, you are also able to add a guest named `Gia`)
```

Comment:
I think this might be a little misleading .. as it gives the impression that the same 'object'/'card' will be in both guest and vendor list (e.g. editing the tag for the guest Gia will also edit it for the vendor Gia).

```suggestion
* A guest and vendor can share the same name (e.g. Even if there is a vendor named `Gia`, you are also able to add a guest named `Gia`)
```